bittide/src/Bittide/SharedTypes.hs

```diff
@@ -158,3 +161,7 @@ forceReset force = unsafeFromHighPolarity (unsafeToHighPolarity hasReset .||. fo
 -- | Divide and round up.
 divRU :: Integral a => a -> a -> a
 divRU b a = (b + a - 1) `div` a
+
+-- | Sort a vector using 'Data.List.sortOn'.
+sortVec :: (KnownNat n, Ord b) => (a -> b) -> Vec n a -> Vec n a
+sortVec f v = unsafeFromList (L.sortOn f (toList v))
```

Comment:
I guess this won't be able to translate to HDL because it relies on `Data.List.sort`? For Haskell-simulation this should work fine, but maybe it would be nice to have a `Clash.Sized.Vec.sort` function eventually

Reply:
Unfortunately, you're right.
I'm working on it as we speak.